content/realtime/channels.textile

```diff
@@ -777,6 +845,16 @@ bc[swift]. realtime.channels.get("private:chatroom").attach { error in
     }
 }
 
+bc[flutter]. Channel channel = realtime.channels.get("private:chatroom");
```

Comment:
It is better not to mix use of `"` and `'` in the same code samples, unless there is a compelling reason.  I note above you use `.split('.')` so not a strictly interpolation need I assume.

Reply:
Oh, thanks. I missed this.
All dart code samples use `'`

Reply:
addressed in e288898cb0f972b74350bd69c2da054346fbb4f0